### PR TITLE
docs: Create Swagger file

### LINKED ISSUE #61: Create Swagger for REST API
Create Swagger file similar files found here: https://github.com/edgexfoundry/edgex-go/tree/main/openapi/v3

### PR DESCRIPTION
**Pull request by lenny-intel**
Also updated the Postman collect for latest used to test.

closes #61

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-record-replay/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?) **N/A*
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?) **Not Yet**
  <link to docs PR>

## Testing Instructions
copy and past Swagger YAML to https://editor.swagger.io/
Verify no parsing errors
Verify each endpoint renders appropriately

### CODE REVIEW

File: Record and Reply.postman_collection.json [hunk 1]
```diff
@@ -51,42 +31,10 @@
 						"v3",
 						"record"
 					]
-				}
+				},
+				"description": "Start Recording starts a recording based on parameters in the reequest body"
```
Comment (jim-wang-intel) [inline Record and Reply.postman_collection.json:35]: typo : 
```suggestion
				"description": "Start Recording starts a recording based on parameters in the request body"
```

File: Record and Reply.postman_collection.json [hunk 2]
```diff
@@ -228,15 +164,76 @@
 						"v3",
 						"data"
 					]
-				}
+				},
+				"description": "Export Recording exports the recorded data along with all referenced devices and device profiles. Data can be exported in plain text JSON, GZIP compresed JSON or ZLIB compressed JSON"
```
Comment (jim-wang-intel) [inline Record and Reply.postman_collection.json:168]: typo: 
```suggestion
				"description": "Export Recording exports the recorded data along with all referenced devices and device profiles. Data can be exported in plain text JSON, GZIP compressed JSON or ZLIB compressed JSON"
```

**Review by jim-wang-intel — COMMENTED** [review]
LGTM except a couple of typos

File: Record and Reply.postman_collection.json (hunk 2 above)
Comment (lenny-intel) [reply Record and Reply.postman_collection.json]: fixed

File: Record and Reply.postman_collection.json (hunk 1 above)
Comment (lenny-intel) [reply Record and Reply.postman_collection.json]: fixed

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Review by seanohair22 — COMMENTED** [review]
Reviewed, LGTM!

**Pull request merged**